components/AppBar/tests/unit/AppBarWrappingLegacyTopLayoutGuideTests.swift

```diff
@@ -0,0 +1,151 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import XCTest
+import MaterialComponents.MaterialAppBar
+import MaterialComponents.MaterialUIMetrics
+
+class AppBarWrappingLegacyTopLayoutGuideTests: XCTestCase {
+
+  // MARK: No scroll view
+
+  func testNoScrollViewTopLayoutGuideEqualsZero() {
+    // Given
+    let contentViewController = UIViewController()
+
+    let container = MDCAppBarContainerViewController(contentViewController: contentViewController)
+    container.isTopLayoutGuideAdjustmentEnabled = false
+    container.appBar.headerViewController.headerView.minMaxHeightIncludesSafeArea = false
```

Comment:
Any reason we're setting `minMaxHeightIncludesSafeArea` to `false` here? Does it break the behavior if it's `true`?

Reply:
minMaxHeightIncludesSafeArea is recommended to be set to false and is the desired behavior of the component, but setting this does not adversely affect the tests.